html/elements/link.json

```
@@ -780,6 +780,42 @@
               }
             }
           },
+          "compression-dictionary": {
+            "__compat": {
+              "description": "rel=compression-dictionary",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/link#compression-dictionary",
+              "spec_url": "https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-compression-dictionary#name-link-relation-registration",
+              "support": {
+                "chrome": {
+                  "version_added": "130"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "5"
+                },
```

How did you determine that Safari 5 started to support this?

---

Oh it definitely does not support this! Copy and paste error. Good spot!

```suggestion
                "safari": {
                  "version_added": false
                },
```